src/Arriba/Arriba.Composition/Composition/Host.cs

```diff
@@ -48,6 +48,7 @@ public Host()
             };
 
             _configuration = new ContainerConfiguration().WithAssemblies(assemblies.Distinct(), conventions);
+            Compose();
```

Comment:
This doesn't feel like the correct change, however I'll approve anyhow as I have a change coming that unifies the composition models and gets rid of this class all together